Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -815,7 +815,7 @@ private void HandleManipulationStarted()
                 wasGravity = rigidBody.useGravity;
                 wasKinematic = rigidBody.isKinematic;
                 rigidBody.useGravity = false;
-                rigidBody.isKinematic = false;
+                rigidBody.isKinematic = true;
```

Comment:
Unfortunately, this causes the scenario to break for far manipulation.

If you have time, can you try casing using the isNearManipulation flag? After making the changes, if it passes our unit tests we can revisit merging this fix.